### PR TITLE
Add option to include other update images

### PR DESCRIPTION
**Pull request by VladimirSlavik**
With this, the Anaconda updates script can now merge other images, such as from add-ons in development.

### CODE REVIEW

File: anaconda_updates/update_image.py [hunk 1]
```diff
@@ -145,6 +152,26 @@ def prepare(self):
...
+        if GlobalSettings.add_image:
+            try:
+                os.makedirs(image_img_path)
+            except os.error as e:
+                print("Can't create image directory!")
+
+        for image in GlobalSettings.add_image:
+            print("Add image", image)
+            cmd = "gunzip < {} | cpio --quiet -iduD {}".format(image, image_img_path)
+            popen = subprocess.Popen(cmd, shell=True)
```
Comment (jkonecny12) [inline anaconda_updates/update_image.py]: Please use [subprocess.run()](https://docs.python.org/3/library/subprocess.html#subprocess.run) here.

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
Other than this it looks good to me.

File: anaconda_updates/update_image.py (hunk 1 above)
Comment (VladimirSlavik) [reply anaconda_updates/update_image.py]: Solved!

**Review by jkonecny12 — APPROVED** [review]
Looks good to me now.

**Comment by VladimirSlavik** [comment]
Thank you!

**Pull request merged**